interaction model/newSchema.json

```diff
@@ -0,0 +1,416 @@
+{
+  "interactionModel": {
+    "languageModel": {
+      "invocationName": "One bus away",
```

Comment:
This has to be `one bus away` to keep the invocation name the same - the Alexa console won't build the model with the capital `O`.

Reply:
Thank you so much for the quick response,
modified to `one bus away`